src/main/java/uk/gov/hmcts/ccd/util/ClientContextUtil.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.ccd.util;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+public class ClientContextUtil {
+    public static Logger LOG = LoggerFactory.getLogger(ClientContextUtil.class);
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    public static String mergeClientContexts(String jsonA, String jsonB) {
+        ObjectNode nodeA = null;
+        ObjectNode nodeB = null;
+
+        try {
+            nodeA = (ObjectNode) objectMapper.readTree(jsonA);
+        } catch (IOException e) {
+            LOG.error("Problem deserialising jsonA: {}", jsonA, e);
+        } catch (Exception e) {
+            LOG.error("Problem with jsonA: {}", jsonA, e);
+        }
+        try {
+            nodeB = (ObjectNode) objectMapper.readTree(jsonB);
+        } catch (IOException e) {
+            LOG.error("Problem deserialising jsonB: {}", jsonB, e);
+        } catch (Exception e) {
+            LOG.error("Problem with jsonB: {}", jsonB, e);
+        }
+
+        if (null == nodeA && null == nodeB) {
+            return jsonA;
+        } else if (null == nodeB) {
+            return jsonA;
+        } else if (null == nodeA) {
+            return jsonB;
+        }
+
+        mergeObjectNodes(nodeA, nodeB);
+        String mergedJson = "";
+        try {
+            mergedJson = objectMapper.writeValueAsString(nodeA);
+        } catch (IOException e) {
+            LOG.error("Problem deserialising mergedJson: {}", mergedJson, e);
+        }
+        return mergedJson;
```

Comment:
This can be safely simplified:
```suggestion
        return nodeA.toString();
```